conf/ref_databases.config

```diff
@@ -323,6 +323,12 @@ params {
             dbversion = "Zehr-nifH v. 2.5.0"
             taxlevels = "Domain,Phylum,Class,Order,Family,Genus"
         }
+        'plantae-bold-trnl' {
```

Comment:
There must be versioning. What BOLD database version is that based on? This would be a good starting point. 

The aim is to have two entries, the general one which will always link to the most recent version, i.e. `'plantae-bold-trnl'` and a specific version, e.g. `'plantae-bold-trnl=6.0'` that will allow adding newer versions while keeping the older versions available.

Reply:
Thanks for the clarification! Maybe it's correct now?

Reply:
Done and done. I also added one more database for ITS1 marker, following the previous indications.